server/sonar-web/src/main/js/apps/projects/query.ts

```diff
@@ -170,23 +170,19 @@ function getVisualization(value: string): string | undefined {
 }
 
 function convertIssuesRating(metric: string, rating: number): string {
-  if (rating > 1 && rating < 5) {
-    return `${metric} >= ${rating}`;
-  } else {
-    return `${metric} = ${rating}`;
-  }
+  return `${metric} = ${rating}`;
 }
 
 function convertCoverage(metric: string, coverage: number): string {
   switch (coverage) {
     case 1:
       return metric + ' >= 80';
     case 2:
-      return metric + ' < 80';
+      return `${metric} < 80 AND ${metric} >= 70`;
     case 3:
-      return metric + ' < 70';
+      return `${metric} < 70 AND ${metric} >= 50`;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use a template literal instead of concatenating with a string literal. ([typescript:S3512](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3512))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3141&issues=AWIKRrzD54IFfP4XjNBx&open=AWIKRrzD54IFfP4XjNBx)

[ISSUE_KEY]: AWIKRrzD54IFfP4XjNBx